Sources/Turf/Geometries/Polygon.swift

```diff
@@ -298,27 +298,42 @@ extension Polygon {
         )
     }
 
+    /// Calculates the absolute centre (of the bounding box).
+    public var center: LocationCoordinate2D? {
+        // This implementation is a port of: https://github.com/Turfjs/turf/blob/master/packages/turf-center/index.ts
+        return BoundingBox(from: outerRing.coordinates)
+            .map { .init(
+                latitude: ($0.southWest.latitude + $0.northEast.latitude) / 2,
+                longitude: ($0.southWest.longitude + $0.northEast.longitude) / 2
+            ) }
+    }
 
     /// Calculates the centroid using the mean of all vertices.
     /// This lessens the effect of small islands and artifacts when calculating the centroid of a set of polygons.
-    public func centroid() -> LocationCoordinate2D? {
+    public var centroid: LocationCoordinate2D? {
+        // This implementation is a port of: https://github.com/Turfjs/turf/blob/master/packages/turf-centroid/index.ts
+        
         let coordinates = outerRing.coordinates.dropLast()
+        guard coordinates.count > 0 else { return nil }
+        
         let summed = coordinates
             .reduce(into: LocationCoordinate2D(latitude: 0, longitude: 0)) { acc, next in
                 acc.latitude += next.latitude
                 acc.longitude += next.longitude
             }
-        return .init(
+        return LocationCoordinate2D(
             latitude: summed.latitude / Double(coordinates.count),
             longitude: summed.longitude / Double(coordinates.count)
-        )
+        ).normalized
```

Comment:
I was hasty in suggesting “wrapping” above: it’s more correct than snapping but maybe not what a developer would expect. If a feature (such as Alaska) straddles the antimeridian, I think this will cause the bounding box to include everything except Alaska at the same latitudes, effectively going the longer way around the world. It could be that your original code worked fine, but a test would give us more confidence. If not, I think it’s OK to restore the original version (which matches Turf.js more closely) and track this edge case separately, as with #118.

Reply:
Oh, my mistake, `normalized` is being applied to the final result, not each individual coordinate. So this will do the right thing. Sorry for the confusion.